### PR TITLE
WIFI 792 - Adding RF profile parameters to static and setups

### PR DESCRIPTION
**Pull request by ralphlee3**
Adding the RF profile plumbing for all parameters, updating static and setups

### CODE REVIEW

File: profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java [hunk 1]
```diff
@@ -19,12 +24,12 @@
 	
     private static final Logger LOG = LoggerFactory.getLogger(RfElementConfiguration.class);
     
-//  private final static int MIN_TWOFOURG_RADIO_CELL_SIZE = -80;
-//	private final static int MIN_FIVEG_RADIO_CELL_SIZE = -80;
-//	private final static int MIN_FIVEGUPPER_RADIO_CELL_SIZE = -80;
-//	private final static int MIN_FIVEGLOWER_RADIO_CELL_SIZE = -80;
-//	public final static int DEFAULT_RX_CELL_SIZE_DB = -90;
-//	public final static int DEFAULT_EIRP_TX_POWER = 18;
+    private final static int MIN_TWOFOURG_RADIO_CELL_SIZE = -80;
```
Comment (christieunitt-cu) [inline profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java:27]: Can the MIN constants be moved into a map per radioType. We will only have more of them (eg 6G)

File: profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java [hunk 2]
```diff
@@ -60,28 +66,34 @@ private RfElementConfiguration() {
...
+    	setNeighbouringListApConfig(NeighbouringAPListConfiguration.createDefault());
+    	setPerimeterDetectionEnabled(true);
+    	setChannelHopSettings(ChannelHopSettings.createWithDefaults());
     }
     
     public static RfElementConfiguration createWithDefaults(RadioType radioType) {
         RfElementConfiguration ret = new RfElementConfiguration();  
-//      ret.setBestApSettings(RadioBestApSettings.createWithDefaults(radioType));
+        ret.setBestApSettings(RadioBestApSettings.createWithDefaults(radioType));
         if (radioType == RadioType.is5GHz || radioType == RadioType.is5GHzL || radioType == RadioType.is5GHzU) {
```
Comment (christieunitt-cu) [inline profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java:85]: I suggest adding a new boolean method to RadioType to return true if it's param is any type of 5G. We'll have other cases which will need this kind of check eg boolean is5gFrequency(RadioType radioType). Someone else might have a better name.

File: profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java (hunk 1 above)
Comment (ralphlee3) [reply profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java:27]: I can make that adjustment

File: profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java (hunk 2 above)
Comment (ralphlee3) [reply profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java:94]: Maybe I should have done the reverse to simplify the logic, to check if it's 2.4G to set 20MHz, else set 80MHz. 

But if we are to support more radios in the future, all the else cases for RadioType will need to be refactored as well, not just here.

**Pull request merged**